tests/test_models.py

```diff
@@ -711,12 +711,11 @@ def test_branding_configuration_saving_successfully(self):
             self.assertEqual(EnterpriseCustomerBrandingConfiguration.objects.count(), 2)
 
     @ddt.data(
-        (False, 350 * 1024),
-        (False, 251 * 1024),
-        (False, 250 * 1024),
-        (False, 4 * 1024),
-        (True, 2 * 1024),
-        (True, 3 * 1024),
+        (False, 2048 * 1024),
```

Comment:
Rather than needing to change this test, let's actually add the settings to `test_settings.py` and rely on that configuration remaining the same. Either that or use the `@override_settings` decorator.